packages/ia-components/sandbox/bookreader-component/bookreader-wrapper-main.jsx

```diff
@@ -0,0 +1,64 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+
+/**
+ * BookReaderWrapper wraps globally accessible BookReader to create an instance
+ *
+ * This component is client-side only.
+ * This component expects all necessary scripts for BookReader to be pre-loaded onto the page before use.
+ * Please see https://github.com/internetarchive/bookreader for more instructions & examples.
+ *
+ * global: BookReader
+ * Creates a global: window.br (which Search will need)
+ *
+ */
+export default class BookReaderWrapper extends Component {
+  constructor(props) {
+    super(props);
+    this.BookReaderRef = React.createRef();
+  }
+
+  componentDidMount() {
+    const { options } = this.props;
+    const defaultOptions = {
+      el: `#${this.BookReaderRef.current.id}`,
+      mobileNavFullscreenOnly: true,
+      onePage: { autofit: 'height' }, // options: auto, width, height
+      ui: 'full',
+      enablePageResume: false,
+      enableTtsPlugin: false,
+      enableUrlPlugin: false,
+      defaults: 'mode/1up',
+      enableSearch: true,
+      searchInsideUrl: '/fulltext/inside.php',
+      initialSearchTerm: null,
+      imagesBaseURL: '/bookreader/BookReader/images/',
+    };
+
+    const fullOptions = {
+      ...defaultOptions,
+      ...options,
+      defaultStartLeaf: 0,
```

Comment:
Why aren't these two properties in the `defaultOptions`? Are we intentionally overriding these props no matter what comes in?

Reply:
Yes, we are overriding it until we can pass the correct defaults for audio liner notes through the backend service